README.md

```diff
@@ -1 +1,82 @@
-# Rakugo
\ No newline at end of file
+# ![Logo](graphics/window_icon.png) Rakugo
+
+![MIT License](https://img.shields.io/static/v1.svg?label=📜%20License&message=MIT&color=informational)
+[![Join the Discord channel](https://img.shields.io/static/v1.svg?label=Join%20our%20Discord%20channel&message=🎆&color=7289DA&logo=discord&logoColor=white&labelColor=2C2F33)](https://discord.gg/K9gvjdg)
+[![GitHub](https://img.shields.io/github/contributors/rakugoteam/Rakugo.svg)](https://github.com/rakugoteam/Rakugo)
+[![GitHub](https://img.shields.io/github/stars/rakugoteam/Rakugo.svg)](https://github.com/rakugoteam/Rakugo)
+[![GitHub](https://img.shields.io/github/forks/rakugoteam/Rakugo.svg)](https://github.com/rakugoteam/Rakugo/network)
+[![github-watchers](https://img.shields.io/github/watchers/rakugoteam/Rakugo?label=Watch&style=social&logo=github)](https://github.com/rakugoteam/Rakugo)
+[![GitHub](https://img.shields.io/github/issues/rakugoteam/Rakugo.svg)](https://github.com/rakugoteam/Rakugo/issues)
+[![GitHub](https://img.shields.io/github/issues-closed/rakugoteam/Rakugo.svg)](https://github.com/rakugoteam/Rakugo/issues)
+
+Inspired by [Ren'Py](https://www.renpy.org), Rakugo is a project aiming to provide a way to make narrative-based games on [Godot](https://godotengine.org) easily. Simplify your project, if it is a visual novel, point and click, RPG, interactive text game or many other styles and blends of styles.
+
+Support this project here [itch.io](https://rakugoteam.github.io/donations/).
+
+[Showcase](#Showcase) -
+[Installation](#installation) -
+[Rakugo Addons](#Rakugo Addons) -
+[Roadmap](#Roadmap) -
+[FAQ](#faq) -
+[Infos](#Infos)
+
+---
+
+## Feature
+* Save/Load system
+* Funcs, Objects and Vars like in Ren'Py
+* Documentation: https://rakugodocs.readthedocs.io
+
+## Showcase
+
+- [**Space drive beats**](https://plopsis.itch.io/space-drive-beats)
+- [**Bot Saves Dream**](https://plopsis.itch.io/curator-bot)
+
+If your game uses Rakugo, tell us on [Discord](https://discord.gg/K9gvjdg).
+
+## Installation
+
+To install Rakugo plugin, download it as a [ZIP archive](). Extract the `addons/Rakugo` folder into your project folder. Then, enable the plugin in project settings and restart Godot-Engine.
+
+If you want to know more about installing plugins you can read the [official documentation page](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/installing_plugins.html).
+
+## Rakugo Addons
```

Comment:
Missing https://github.com/rakugoteam/AdvancedText and links to each addons repos